### PR TITLE
Replace databes

### PR DESCRIPTION
**Pull request by gogo2245**

### CODE REVIEW

File: databes.py [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+        self.input_data = []
+        self.targets = []
+        while len(labels) != 0:
+            input_data = []
+            target = []
+            for i in range(5):
+                if (len(labels) == 0):
+                    break
+                rand_label = random.choice(list(labels.keys()))
+                rand = labels[rand_label]
```
Comment (martinPasen) [inline databes.py:30]: are lines 29 and 30 equivalent to 
`rand = random(len(labels))`
or is there happening something more complex?

File: databes.py [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+                bytes, _, cur = data[rand]
+                input_data += bytes[cur * 300:(cur + 1) * 300]
+                target += [rand]*300
```
Comment (martinPasen) [inline databes.py:33]: what if the length of bytes isn't divisible by 300. Can't we get input_data shorter and therefore target have the same length? Later in training, it will probably crash, if our output won't have the same length as targets.

File: databes.py (hunk 1 above)
Comment (gogo2245) [reply databes.py:30]: No this is done this way because when some dataset is exhausted We remove it from labels.

**Comment by gogo2245** [comment]
I am thinking now could the issue we discussed last time be float overflow?

**Comment by gogo2245** [comment]
Found this: https://discuss.pytorch.org/t/nan-loss-coming-after-some-time/11568

**Comment by gogo2245** [comment]
It seems this can be resolved with gradient clipping

**Review by martinPasen — APPROVED** [review]
